--- src/qterminal/src/termwidget.cpp	2016-06-21 00:27:51.707809909 -0700
+++ src/qterminal/src/termwidget.cpp-new	2016-06-21 01:02:28.620164898 -0700
@@ -104,20 +104,7 @@
         setScrollBarPosition(QTermWidget::ScrollBarRight);
         break;
     }
-
-    switch(Properties::Instance()->keyboardCursorShape) {
-    case 1:
-        setKeyboardCursorShape(QTermWidget::UnderlineCursor);
-        break;
-    case 2:
-        setKeyboardCursorShape(QTermWidget::IBeamCursor);
-        break;
-    default:
-    case 0:
-        setKeyboardCursorShape(QTermWidget::BlockCursor);
-        break;
-    }
-
+   
     update();
 }
 
